src/main/kotlin/org/koil/org/Models.kt

```diff
@@ -0,0 +1,22 @@
+package org.koil.org
+
+import org.koil.user.Account
+import org.springframework.data.annotation.Id
+import org.springframework.data.relational.core.mapping.Table
+import java.time.Instant
+import java.util.*
+
+@Table("organization")
+data class Organization(
+    @Id val organizationId: Long?,
+    val organizationName: String,
+    val startDate: Instant,
+    val stopDate: Instant?,
+    val signupLink: UUID
+)
+
+sealed class OrgAccountUpdateResult {
+    data class AccountUpdateSuccess(val account: Account) : OrgAccountUpdateResult()
+    data class EmailAlreadyTaken(val account: Account) : OrgAccountUpdateResult()
```

Comment:
Yes- because you are updating an Organization's Account (e.g. a User or individual's account). This doesn't update the Organization itself. This is almost an exact duplicate of the AccountUpdate request/response that exists for an Admin - however there are differences between what is available for an Admin Account Update and an OrgAccountUpdate.

Reply:
That makes sense.